tests/integration/test_sqlalchemy_integration.py

```diff
@@ -295,3 +296,26 @@ def test_json_column(trino_connection, json_object):
         assert rows[0] == (1, json_object)
     finally:
         metadata.drop_all(engine)
+
+
+@pytest.mark.parametrize('trino_connection', ['memory'], indirect=True)
+def test_get_table_comment(trino_connection):
+    engine, conn = trino_connection
+
+    if not engine.dialect.has_schema(engine, "test"):
+        engine.execute(sqla.schema.CreateSchema("test"))
+    metadata = sqla.MetaData()
+
+    try:
+        sqla.Table(
+            'table_with_id',
+            metadata,
+            sqla.Column('id', sqla.Integer),
+            schema="test"
+        )
+        metadata.create_all(engine)
+        insp = inspect(engine)
+        actual = insp.get_table_comment(table_name='table_with_id', schema="test")
+        assert actual['text'] is None
```

Comment:
It would be great, but it seems to not work to add a comment through sqlalchemy. Let's create a bug for that.

Reply:
create an issue + add a TODO. Goal is to make sure if someone other than you works on that issue then they can easily find places in code which need updating otherwise they may end up adding duplicated tests.